src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Wrapper, List, NavLink } from './styled'
+
+export const Navbar: React.FC = () => {
```

Comment:
I've said that in the overall comment, having multiple navbars seems like a decision that would cost us a lot of users and that would degrade your own work on trying to make buttonNav buttons useful in every situation. Imagine menu on Facebook just disappearing when you click on messages, you'd instead get a new button linking to your page somewhere in the footer. Nobody would be on Facebook rather than to try to remember when certain links change from text to button and when they're where exactly on a page.

Reply:
Ok.